sql/1.107/contributor_activity.sql

```diff
@@ -0,0 +1,12 @@
+BEGIN;
+
+CREATE TABLE contributor_activity (
+    id serial NOT NULL PRIMARY KEY,
+    github_id text NOT NULL UNIQUE,
```

Comment:
@zachthompson i was also struggling with distinguishing the two via their column name. 
`github_id` refers to the github-assigned event id or sha number
`contributor_id` is a foreign key pointing to the user who was responsible for the activity

:100: agree on the cascade issue. will change.